spec/models/user_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+describe User, type: :model do
+  fixtures :users
+  # it 'is valid with all attributes' do
+  #   expect(User.new(name: 'wandji', email: 'wandji@email', password: '111111')).to be_valid
+  # end
+  # it 'is not valid with name longer than 20 characters' do
+  #   expect(User.new(name: 'wandjirrrrrrrrrrttttttt', email: 'wandji@email', password: '111111')).to_not be_valid
+  # end
+  # let(:user) { User.new(name: 'wandjir', email: 'wandji@email', password: '111111')}
```

Comment:
It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))